src/js/modules/core/services/sweeper/sweeper.service.js

```diff
@@ -102,30 +97,58 @@
             }
 
             return bitcoinDataClient.estimateFee().then(function (feePerKB) {
-                if (totalValue <= blocktrailSDK.DUST) {
-                    return false;
-                }
 
-                rawTransaction.addOutput(options.recipient, totalValue);
+                var txAfter = 600;
 
-                var fee = blocktrailSDK.Wallet.estimateIncompleteTxFee(rawTransaction.tx, feePerKB);
-                rawTransaction.tx.outs[0].value -= fee;
+                var chunks = [];
+                var index = 0;
+                var length = inputs.length;
 
-                if (rawTransaction.tx.outs[0].value <= blocktrailSDK.DUST) {
-                    return false;
+                while (index < length) {
+                    chunks.push(inputs.slice(index, index += txAfter + 1));
+                    // index += txAfter;
```

Comment:
the assignment should probably be moved out of `inputs.slice`.. could it be rewritten as a for loop?

Reply:
Yes, that's possible. Done.